openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -239,6 +264,57 @@ def export(self, request, **_kwargs) -> HttpResponse:
 
         return HttpResponse(tags, content_type=content_type)
 
+    @action(detail=False, url_path="import", methods=["post"])
+    def create_import(self, request: Request, **_kwargs) -> HttpResponse:
+        """
+        Create a new taxonomy and imports the tags from the uploaded file.
+        """
+        perm = "oel_tagging.import_taxonomy"
+        if not request.user.has_perm(perm):
+            raise PermissionDenied("You do not have permission to import taxonomies")
```

Comment:
I think we can assume that if a user can create a taxonomy, he also can create a taxonomy via import.
If this is true, we don't need the `import_taxonomy` permission because the POST permission will be called anyway in this endpoint.

Reply:
@rpenido I agree -- simpler is better, so it's ok to rely on the `create` and `change` taxonomy permissions that are checked by the view.

Reply:
Done: ae764004976bcc8801b86a409046c843ae3b4721
I went out of scope and did the same for the `export`. If you can view a taxonomy, you can export it. Let me know if it is not the right time to do that, and I revert it!

Reply:
Cool, thank you for doing the same for export! That's cleaner all around.